apps/mcp-server/src/utils/logging.ts

```diff
@@ -0,0 +1,84 @@
+import type { RequestInfo } from "@modelcontextprotocol/sdk/types.js";
```

Comment:
i don't really like the fact that we are using winston here. 

Pat is working on a centralized logging library. Can we use something like the Console Logger from the widets in the meantime? copy the logger here?